torch/csrc/api/include/torch/optim/sgd.h

```diff
@@ -48,7 +49,7 @@ class TORCH_API SGD : public Optimizer {
   SGD() : options(0) {}
 
   /// Counts how often `step()` is called, for dampening.
-  size_t iteration_{0};
+  int64_t iteration_{0};
```

Comment:
This is changed from `size_t` to `int64_t`, because `IValue` doesn't support storing a `size_t` value.